modules/odc-viewing-application-topology.adoc

```diff
@@ -24,7 +19,16 @@ image::odc_application_topology.png[]
 
 The application resource name is appended with indicators for the different types of resource objects as follows:
 
-* *DC*: DeploymentConfigs
+* *CJ*: CronJob
 * *D*: Deployment
-* *SS*: StatefulSet
+* *DC*: DeploymentConfigs
 * *DS*: Daemonset
+* *J*: Job
+* *P*: Pod
+* *SS*: StatefulSet
```

Comment:
Can you please confirm what exactly the long forms should be? As per my understanding, CJ:CronJobs, P:Pods, SS:StatefulSets, J:Jobs and D:Deployments. If what you have added is right, then it looks good to me. Thanks!

Reply:
@gajanan-more let's use signular, because that's what the abbrev in the UI actually stands for, wdyt?

Reply:
I am okay with it. Thanks!!!